keywrap/jwe/keywrapper_jwe.go

```diff
@@ -65,7 +65,11 @@ func (kw *jweKeyWrapper) WrapKeys(ec *config.EncryptConfig, optsData []byte) ([]
 }
 
 func (kw *jweKeyWrapper) UnwrapKey(dc *config.DecryptConfig, jweString []byte) ([]byte, error) {
-	jwe, err := jose.ParseEncrypted(string(jweString))
+	// cf. list of algorithms in func addPubKeys() below
+	key_algorithms := []jose.KeyAlgorithm{jose.RSA_OAEP, jose.RSA_OAEP_256, jose.ECDH_ES_A128KW, jose.ECDH_ES_A192KW, jose.ECDH_ES_A256KW}
```

Comment:
Looks good. This happens to be the same list as this one here: 

https://github.com/containers/ocicrypt/blob/30c2fb6a4b668c23038a290d64c1f9b7c686f170/keywrap/jwe/keywrapper_jwe.go#L129-L142